mps_youtube/commands/download.py

```diff
@@ -332,6 +332,21 @@ def list_string_sub(orig, repl, lst):
     util.dbg("Downloading using: %s", " ".join(cmd_list))
     subprocess.call(cmd_list)
 
+def _transcode(song, filename, audio=False, allow_transcode=True):
```

Comment:
taking `allow_transcode` in transcode function does not make sense. Please remove this, we don't pass it anyway.

Reply:
fixed in the next commit